<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>effect</class>
 <widget class="QWidget" name="effect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>圖型效果DEMO</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="Taiwan"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>4</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="locale">
      <locale language="Chinese" country="Taiwan"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="toolBoxPage1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>120</width>
        <height>175</height>
       </rect>
      </property>
      <attribute name="label">
       <string>模糊效果</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>半徑:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="radiusSpin">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>117</width>
        <height>175</height>
       </rect>
      </property>
      <attribute name="label">
       <string>著色效果</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="redRadio">
         <property name="text">
          <string>紅色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="greenRadio">
         <property name="text">
          <string>綠色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="blueRadio">
         <property name="text">
          <string>藍色</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>濃度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="strengthSpin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-12</y>
        <width>126</width>
        <height>187</height>
       </rect>
      </property>
      <attribute name="label">
       <string>陰影效果</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="shadowRedRadio">
         <property name="text">
          <string>紅色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="shadowGreenRadio">
         <property name="text">
          <string>綠色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="shadowBlueRadio">
         <property name="text">
          <string>藍色</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>半徑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="shadowRSpin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>X偏移</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xOffsetSpin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>50.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>8.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Y偏移</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="yOffsetSpin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>50.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>8.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="toolBoxPage2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>130</width>
        <height>175</height>
       </rect>
      </property>
      <attribute name="label">
       <string>透明效果</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>透明度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="opacitySpin">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.700000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
